### DISCUSSION TITLE
Unify terminology for the "Editor"

### DISCUSSION THREAD

**Discussion by decryptedchaos** [stride3d/stride#1998] [Polls]
As discussed briefly in discord,  I am targeting the next contrib meeting with this one.  

Currently,  the Editor is officially called  "Stride Game Studio" this seems out of sync with most of the industry with the term "Editor" being mostly used. 

Even within our own internal discussions and notes, the term "Editor" is almost exclusively used,  This is a consistency issue,  We should probably unify the term used for not only UX but also documentation reasons.  

As I say, I would like to bring this to the table officially in the next meeting..  However so that discussion and debate can take place beforehand I propose it here.

**Comment by manio143** [comment]
I have been generally using the phrase GameStudio because this is what the C# project is called which is used to launch the editor experience. There may be many editors within the GameStudio. But I guess nothing prevents us from renaming it - just need to be conscious this may be a major breaking change.

**Comment by decryptedchaos** [comment]
##### *Copied from my discord reply*:

I understand the technical complications,  i would say however that the longer it goes forward the harder it will be to change..  So,  my issue comes from one of human perspective,  i know a few things about how humans statistically judge things..  things that are too different often trigger subconscious mistrust or doubt when it comes to something someone is going to invest time into..    

To a technical designer  the term is ambiguous,   but in a general sense,  there are norms in software that the general masses expect,  straying from those norms too much can and often does cause people again statistically to believe the overall project does not follow traditional conventions..

To expand on that a bit, this is most often a problem in niche applications that require a lot of skill   like graphic design software,  3D modelling,  /  CAD   and of course game engines.   interestingly,  normal everyday users are more willing to accept minor changes to simple applications like email for example  than high skilled app users would be.
